src/middlewared/middlewared/plugins/jail.py

```diff
@@ -485,8 +493,6 @@ def progress_callback(content, exception):
         iocage.fetch(**options)
 
         if post_install and options['name'] is not None:
-            pool = IOCJson().json_get_value('pool')
-            iocroot = IOCJson(pool).json_get_value('iocroot')
             plugin_manifest = pathlib.Path(
                 f'{iocroot}/.plugin_index/{options["name"]}.json'
```

Comment:
`iocroot` would be undeclared here ?

Reply:
No, it's declared above

Reply:
that is in a different block ?

Reply:
Which is always triggered before this ;)

Reply:
actually nope, the if will work ; )

Reply:
Do you think it might be a better idea to declare them out of the ifs though just for future cases where we might introduce changes removing this in the first if and mistakingly forget about the usage below ?